static/src/javascripts/projects/common/modules/identity/manage-account.js

```diff
@@ -37,17 +39,61 @@ const submitPartialFormStatus = (
     });
 };
 
+const getNewsletterHtmlPreferenceFromElement = (
+    originalEl: HTMLElement
+): Promise<string> =>
+    fastdom.read(() => {
+        const closestFormEl: ?Element = originalEl.closest('form');
+
+        if (!closestFormEl) throw new Error(ERR_IDENTITY_HTML_PREF_NOT_FOUND);
+
+        const inputEls: Array<any> = [
```

Comment:
@gustavpursche @SiAdcock not a huge fan of this `<any>` here but was banging my head against flow for a while until I gave up :(. 

Is there a way to enforce `querySelectorAll()` to only return `HTMLInputElement` ? I saw people casting it to `any` then to `HTMLInputElement` which sounds even worse and i tried filtering the array but errors started getting extremely cryptic at that point

Reply:
filtering is like extremely clean and elegant and i love it but i'm not quite sure what flow wants from me

![screen shot 2017-11-23 at 13 09 10](https://user-images.githubusercontent.com/11539094/33174343-8d914b50-d04f-11e7-9d5b-0c95ab1d740e.png)

Reply:
AHAHA NEVERMIND GOT IT